integration/js/app/package.json

```diff
@@ -13,12 +13,12 @@
     "html-webpack-plugin": "^5.3.2",
     "ts-loader": "^9.1.1",
     "typescript": "^4.2.4",
-    "webpack": "^5.36.2",
-    "webpack-cli": "^4.6.0"
+    "webpack": "^5.89.0",
+    "webpack-cli": "^4.8.0"
```

Comment:
need to update the webpack version to avoid this issue:
https://github.com/webpack/webpack/issues/14532

The webpack hashing algorithm uses `node:crypto`